docs/fides/docs/tutorial/index.md

```diff
@@ -0,0 +1,87 @@
+# Tutorial Overview
+
+In this tutorial you will learn how to use the Fidesctl open-source developer tools to solve a real-world data privacy problem. The steps closely follow the commits found in [the `ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo).
+
+You will run a local instance of a basic web app to demonstrate the use of Fidesctl as part of a "real" project that uses:
+
+* Flask to run a web server simulating the basic e-commerce application
+* PostgreSQL as the application's database
+* SQLAlchemy to connect to the database
+* [`fidesctl`](https://github.com/ethyca/fides) to declare privacy manifests and evaluate policies
+
+The app itself is simply the [Flask tutorial app](https://flask.palletsprojects.com/en/2.0.x/tutorial/), but modified to simulate an e-commerce marketplace. This helps to highlight some basic examples of data categories that might be stored in a "real" user-facing application.
+
+## Setup Instructions
+
+### System Requirements
+
+Before beginning, ensure you have the following software installed and configured to your liking:
```

Comment:
Check the latest version of fidesdemo, I also explictly added the system requirement to have `pg_config` installed